ion-schema/src/model/type_definition.rs

```diff
@@ -46,7 +49,8 @@ impl TypeDefinition {
 /// A version-safe builder for Ion Schema type definitions.
 #[derive(Debug, Clone)]
 pub struct TypeDefinitionBuilder<V: IslVersion> {
-    pub(crate) constraints: Vec<AnyConstraint>,
+    constraints: Vec<AnyConstraint>,
+    // FIXME: Should probably be (String, IonData<Element>)
     open_content: Vec<(Symbol, IonData<Element>)>,
```

Comment:
🗺️ I left a note here (rather than change it) because I'm not convinced enough yet to change it. Do you have any thoughts on what this should be?

Reply:
Why would it be `String` rather than `Symbol`? (Presumably ISL allows `$0` field names as open content, even though: Yuck.)